src/Pagination/index.js

```diff
@@ -1,61 +1,10 @@
-import React, { useState, useEffect } from "react";
+import React, { useState } from "react";
 import PropTypes from "prop-types";
 import cc from "classcat";
 import Row from "../Row";
+import { usePagination } from "./usePagination";
 
 const noop = () => {};
-const MAX_VISIBLE_PAGES = 5;
-
-/**
- * Business logic for pagination; given a total number of pages
- * and the selected page, returns attributes used to render pagination
- *
- * @param {Number} totalPages
- * @param {Number} selectedPage
- * @returns {Object} attributes used to render pagination
- */
-export const _getAttributes = (
-  totalPages,
-  selectedPage,
-  segmentSize = MAX_VISIBLE_PAGES
-) => {
-  // create a list of page segments
-  // each segment is an array of page numbers.
-  const pageSegments = [...new Array(totalPages)]
-    .map((p, i) => i + 1) // [1,2,3,4,5,6,7, ...]
```

Comment:
You'll notice this trick being used in my `range` implementation in the usePagination hook.
It's wild that JS still doesn't have native range